include/netutils/telnetd.h

```diff
@@ -77,6 +91,23 @@ extern "C"
 #define EXTERN extern
 #endif
 
+/****************************************************************************
+ * Name: telnetd_daemon
+ *
+ * Description:
+ *   This function is the Telnet daemon.  It does not return (unless an
+ *   error occurs).
+ *
+ * Parameters:
+ *   Standard task start up arguments.
+ *
+ * Return:
+ *   Does not return unless an error occurs.
+ *
+ ****************************************************************************/
+
+int telnetd_daemon(int argc, FAR char *argv[]);
```

Comment:
who will do telnetd_daemon?

Reply:
>who will do telnetd_daemon?

I will create a new PR later where `examples/telnetd/telnetd.c` calls `telnetd_daemon`.

Reply:
I've added `system/telnetd` instead of modifying `examples/telnetd`.